pkg/scalers/aws_sqs_queue_scaler.go

```diff
@@ -25,26 +26,45 @@ const (
 )
 
 type awsSqsQueueScaler struct {
-	metadata *awsSqsQueueMetadata
+	metadata  *awsSqsQueueMetadata
+	sqsClient *sqs.SQS
 }
 
 type awsSqsQueueMetadata struct {
 	targetQueueLength  int
 	queueURL           string
-	region             string
+	awsRegion          string
```

Comment:
Just curious - Why are we renaming this as this is specific for an AWS SQS scaler? 🤔

Reply:
I renamed it to match the AWS_REGION variable name like the other aws variables. Since it's an AWS specific struct I didn't see too much of a problem.